app/controllers/developers/celebration_package_requests_controller.rb

```diff
@@ -18,14 +19,17 @@ def create
 
     private
 
+    def test
+    end
+
```

Comment:
Please delete this code.

Reply:
Removed in d857138a3e4028ed86a75a07aa35b18df5a878f9